srcpkgs/python-pytest/template

```diff
@@ -12,7 +12,7 @@ depends="python-atomicwrites python-attrs python-funcsigs python-more-itertools
 checkdepends="python3-atomicwrites python3-hypothesis python3-more-itertools
  python3-pluggy python3-py python3-atomicwrites"
 short_desc="Simple powerful testing with Python2"
-maintainer="Orphaned <orphaned@voidlinux.eu>"
+maintainer="maxice8 <thinkabit.ukim@gmail.com>"
```

Comment:
i mean fix the email, not re-assign me as maintainer, i have no interest in python maintenance

Reply:
sorry, fixed. I didn't even notice the difference in Orphaned and Orphan....